05-services.md

```diff
@@ -1754,13 +1754,23 @@ parameters (sysctls) at runtime](https://docs.docker.com/engine/reference/comman
 `tmpfs` mounts a temporary file system inside the container. It can be a single value or a list.
 
 ```yml
-tmpfs: /run
+tmpfs: 
+ - <path>:<options>
```

Comment:
as `options` is optional, better document example with and without options

Reply:
Good point @ndeloof, I agree with having examples with & without options